### ISSUE TITLE
Chiffres errones

### ISSUE THREAD

**Issue by pp2382** [oalam/covid-analytics#3]
Bonjour, et merci pour votre travail.

J'ai remarque que vous utilisiez une efficacite de 76% pour l'Astra Zeneca. Ceci est l'efficacite concernant les cas symptomatiques. Etant donne que vous regardez l'effet du vaccin sur les entrees en reanimations, ne serait-il pas plus judicieux d'utiliser l'efficacite sur cette metrique. Dans le cas des hospitalizations, le chiffre est de 88% pour AstraZeneca, et sans doute plus haut pour les entrees en reas.
Je pense que ce chiffre de 88% voir 90% serait plus judicieux.
Si j'ai fait une erreur dans mon raisonnement, merci de me l'indiquer.

**Comment by ZaraBar** [comment]
Merci pour votre remarque tout à fait pertinente. Nous avons hésité, mais avons considéré que cette efficacité risquait d’être amputée à cause de la propagation des variants (cf données d’efficacité en vie réelle). Nous allons y réfléchir...

**Comment by buchdag** [comment]
> mais avons considéré que cette efficacité risquait d’être amputée
 
Êtes-vous développeu.r.se.s ou épidémiologistes ?

> cf données d’efficacité en vie réelle

Sources ?

**Comment by oalam** [comment]
nous avons testé avec des taux d'efficacité plus élevés et cela ne change pas la balance de manière significative. On continue de réfléchir à cet axe d'amélioration.

et pour infos nous sommes un tandem développeur/journaliste

**Comment by buchdag** [comment]
> et pour infos nous sommes un tandem développeur/journaliste

Etant donné ce qu'il exprime et la façon dont il est présenté (par Mediapart), je pense que l'indicateur que vous proposez a de fortes chances d'influencer le choix des gens de se faire vacciner ou non. La validité de cet indicateur et votre méthodologie ont-elles été revues par des médecins, épidémiologistes ou autres scientifiques proches du sujet ?

**Comment by ZaraBar** [comment]
Oui, nous l'avons fait relire par plusieurs scientifiques/experts. Et nous nous sommes également appuyés sur les hypothèses choisies par les experts de l'[EMA ](https://www.ema.europa.eu/en/documents/referral/use-vaxzevria-prevent-covid-19-article-53-procedure-assessment-report_en.pdf)et celle du [Winton Center](https://wintoncentre.maths.cam.ac.uk/news/communicating-potential-benefits-and-harms-astra-zeneca-covid-19-vaccine/)
